OpenKh.Tools.Kh2TextEditor/Views/MainWindow.xaml

```diff
@@ -68,12 +68,16 @@
                         Content="US/EU encoder"
                         IsChecked="{Binding Path=EncodingType, Mode=TwoWay, 
                             Converter={StaticResource enumConverter},
-                            ConverterParameter=European}"/>
+                            ConverterParameter=European}" Margin="0,0,-5,0"/>
```

Comment:
Is this an accidental change?

Reply:
The code was like that when I got it. Weird, it could be one of VS shenanigans.

Reply:
I know it can happen if you drag stuff around the WPF preview thing. Could you make it as it was?

Reply:
On it.

Reply:
Done it.